targets/TARGET_Realtek/TARGET_AMEBA/TARGET_MCU_RTL8195A/analogin_ext.h

```diff
@@ -20,7 +20,9 @@
 extern "C" {
 #endif
 
+#if CONFIG_ADC_EN
```

Comment:
also a fix, should not be part of "adding a new target commit"

Reply:
Ok, I'm going to make a new commit